pkgs/development/web/nodejs/v20.nix

```diff
@@ -55,5 +55,20 @@ buildNodejs {
       url = "https://github.com/nodejs/node/commit/33f6e1ea296cd20366ab94e666b03899a081af94.patch?full_index=1";
       hash = "sha256-aVBMcQlhQeviUQpMIfC988jjDB2BgYzlMYsq+w16mzU=";
     })
+    # Fix builds with shared llhttp, remove in the next point release
+    (fetchpatch2 {
+      url = "https://github.com/nodejs/node/commit/ff3a028f8bf88da70dc79e1d7b7947a8d5a8548a.patch?full_index=1";
+      hash = "sha256-LJcO3RXVPnpbeuD87fiJ260m3BQXNk3+vvZkBMFUz5w=";
+    })
+    (fetchpatch2 {
+      url = "https://github.com/nodejs/node/commit/4454d09e8f7225ec1b576ef86c8705bca63a136c.patch?full_index=1";
+      hash = "sha256-M6eme92cY1dhu1I5/v7Tcd3iSlQi5ZeC48qwLoYj2iA=";
+    })
+    # update tests for nghttp2 1.65
+    (fetchpatch2 {
+      url = "https://github.com/nodejs/node/pull/58293.diff?full_index=1";
```

Comment:
FYI, the procedure for unmerged PRs would be to vendor the patch in nixpkgs.

(My understanding is that you want to replace this with the merged commits once the PR is merged.)